src/workers/notifications/commentMention.ts

```diff
@@ -28,6 +28,10 @@ const worker: NotificationWorker = {
     if (authors.has(data.commentMention.mentionedUserId)) {
       return;
     }
+    const parent = comment.parentId && (await comment.parent);
+    if (data.commentMention.mentionedUserId === parent?.userId) {
```

Comment:
I think all the commenters in the same thread receive a notification not just the parent comment.
So we need to check for that. I suggest looking at the "replied to" notification and see the logic there.